pallets/reserve/src/lib.rs

```diff
@@ -20,19 +20,18 @@
 
 //! A module that is called by the `collective` and is in charge of holding
 //! the company funds.
-
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
do you need that flag? It is normally showing up on top of the `benchmarking.rs` file

Reply:
Indeed not required, removed, Done :+1: